ts/deckoptions/DeckOptionsPage.svelte

```diff
@@ -63,6 +54,16 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     export const audioOptions = {};
     export const addonOptions = {};
     export const advancedOptions = {};
+
+    let rows: number | undefined = undefined;
+    $: {
+        rows = Math.ceil($lineCount / 2) + 1;
+    }
+</script>
+
+<script context="module">
+    import { writable } from "svelte/store";
+    export const lineCount = writable(0);
 </script>
```

Comment:
Checks failing because Prettier wants this all the way on top, but it's a hacky solution anyway - bound to get replaced.